app/models/AtsYearChoice.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json.{JsPath, JsString, JsValue, Json, Reads, Writes}
+import play.api.libs.json.Reads._
+import play.api.libs.functional.syntax._
+
+case class AtsYearChoice(atsType: AtsType, year: Int)
+
+object AtsYearChoice {
```

Comment:
This will just become `implicit val formats = Json.format[AtsYearChoice]` and you can remove the reads and writes below. 

It means the responsibility for formatting a class lies in its own class (The S bit in SOLID)

Reply:
done